### PR TITLE
[Ollama] Update ipex-llm ollama readme to v0.4.6

### PR DESCRIPTION
**Pull request by sgwhat**
### 1. Why the change?

Update ipex-llm ollama readme to v0.4.6.

### 2. User API changes

On Linux, we have the following changes:

2.1 Launch ollama service
```
# before `ollama serve`
export LD_LIBRARY_PATH=.:$LD_LIBRARY_PATH
```

2.2 Run ollama serve
```
# before `ollama run <model name>`
export LD_LIBRARY_PATH=.:$LD_LIBRARY_PATH
source /opt/intel/oneapi/setvars.sh
```

### CODE REVIEW

File: docs/mddocs/Quickstart/ollama_quickstart.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ See the demo of running LLaMA2-7B on Intel Arc GPU below.
 > [!NOTE]
 > `ipex-llm[cpp]==2.2.0b20240826` is consistent with [v0.1.39](https://github.com/ollama/ollama/releases/tag/v0.1.39) of ollama.
```
Comment (rnwang04) [inline docs/mddocs/Quickstart/ollama_quickstart.md:20]: please also update old  version here.

File: docs/mddocs/Quickstart/ollama_quickstart.md [hunk 2]
```diff
@@ -80,6 +80,7 @@ You may launch the Ollama service as below:
   export ZES_ENABLE_SYSMAN=1
   source /opt/intel/oneapi/setvars.sh
   export SYCL_CACHE_PERSISTENT=1
+  export LD_LIBRARY_PATH=.:$LD_LIBRARY_PATH
```
Comment (rnwang04) [inline docs/mddocs/Quickstart/ollama_quickstart.md:83]: why we need this ?

File: docs/mddocs/Quickstart/ollama_quickstart.zh-CN.md [hunk 3]
```diff
@@ -19,7 +19,7 @@
 > [!NOTE]
 > `ipex-llm[cpp]==2.2.0b20240826` 版本与官方 ollama 版本 [v0.1.39](https://github.com/ollama/ollama/releases/tag/v0.1.39) 一致。
```
Comment (rnwang04) [inline docs/mddocs/Quickstart/ollama_quickstart.zh-CN.md:20]: please also update old  version here.

**Review by rnwang04 — APPROVED** [review]
I think we also need to update llama.cpp version. others LGTM.

File: docs/mddocs/Quickstart/ollama_quickstart.md (hunk 2 above)
Comment (rnwang04) [reply docs/mddocs/Quickstart/ollama_quickstart.md:83]: Looks  not very friendly for users. It would be better if we can hidden this for users.

Comment (sgwhat) [reply docs/mddocs/Quickstart/ollama_quickstart.md:83]: This is to link to the shared library when running `ollama`. I can try to hide this by modifying the compilation method.

**Pull request merged**